pkg/cli/serve/serve.go

```diff
@@ -291,6 +291,9 @@ func (o *serveOptions) Run() error {
 			r.HandleFunc("/upgrade_policies", upgradePolicyHandler.ServeUpgradePolicyList)
 			r.HandleFunc("/upgrade_policies/{upgrade_policy_id}", upgradePolicyHandler.ServeUpgradePolicyGet)
 			r.HandleFunc("/upgrade_policies/{upgrade_policy_id}/state", upgradePolicyHandler.ServeUpgradePolicyState)
+			o.logger.Info("Initialising Cluster handlers")
+			clusterHandler := handlers.NewClusterHandler(sdkclient, internalID)
+			r.HandleFunc("/my_cluster", clusterHandler.ServeClusterGet)
```

Comment:
I like the idea here, but my personal preference is for `GET /` to purely reference the cluster object itself. The only reason I say this is because the cluster's OCM pull secret does not have permissions to get other clusters. However, maybe it will someday? 
Maybe we can get Hulk opinions here? 
Do we prefer 
```
			r.HandleFunc("/", clusterHandler.ServeClusterGet)
```
or 
```
			r.HandleFunc("/my_cluster", clusterHandler.ServeClusterGet)
```

Reply:
> the cluster's OCM pull secret does not have permissions to get other clusters
I believe this is true.

> However, maybe it will someday?
That's what I am concerned about. In case the situation changes and we want to get other clusters, the api design may not look nice if we don't want to break the existing api. It may become:
```
"GET /"  -> the current cluster.
"GET /cluster/{cluster-id}" -> other clusters.
```
Thus I am thinking to explicitly specify it is getting `my_cluster`.

Reply:
I definitely get the logic, however we've already built upgrade policies to resolve to 
`get /upgrade_policies`. 
To make things consistent, we would probably have to move it to 
`get /my_cluster/upgrade_policies` 
I've started a discussion in team Hulk about this - we can get their advice on how best to proceed. 
https://redhat-internal.slack.com/archives/C01JJKHC0CF/p1700714500668629

Reply:
As per conversation offline, we will use `/` to reflect my cluster as the scope for this is only for it's own cluster.